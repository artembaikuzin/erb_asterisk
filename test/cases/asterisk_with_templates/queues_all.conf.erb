<%= yield_here :all_queues %>

<%= render 'vars_check' %>
