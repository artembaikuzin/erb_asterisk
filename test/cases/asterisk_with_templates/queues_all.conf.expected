; Yield for :all_queues
operators
